systemstesting/serverPerformanceTest.go

```diff
@@ -0,0 +1,79 @@
+package main
+
```

Comment:
This file attempts to create an easy way we can measure the server performance by checking the API status codes and the response time from the selected API endpoints.